README.md

```diff
@@ -44,3 +44,60 @@ module.exports = {
 ```bash
 yarn jest
 ```
+
+
+## Options
+
+This project uses [cosmiconfig](https://github.com/davidtheclark/cosmiconfig), so you can provide config via:
+* a `jest-runner-eslint` property in your `package.json`
+* a `jest-runner-eslint.config.js` JS file
+* a `.jest-runner-eslintrc` JSON file
+
+
+In `package.json`
+```json
+{
+  "jest-runner-eslint": {
+    "cliOptions": {}
+  }
+}
+```
+
+See https://eslint.org/docs/developer-guide/nodejs-api#cliengine for a full
+set of supported `cliOptions`
+
+#### `fix`
+This is similar to running ESLint with `--fix`
+```json
+{
+  "jest-runner-eslint": {
+    "cliOptions": {
+      "fix": true
+    }
+  }
+}
+```
+
+#### `configFile`
+This is similar to running ESLint with `-c /path/to/config-file`
+```json
+{
+  "jest-runner-eslint": {
+    "cliOptions": {
+      "configFile": "/path/to/config-file"
+    }
+  }
+}
+```
+
+#### `globals`
+This is similar to running ESLint with `--globals`
+```json
+{
+  "jest-runner-eslint": {
+    "cliOptions": {
```

Comment:
@ljharb I wonder if it is even worth having it nested under `cliOptions`?

I originally did it so that it is a bit more feature proof in case we add options that do no map to `cliOptions`, but it might be an overkill

Reply:
Having been burned by this in the past, I think making `cliOptions` is definitely prudent, even if it's a bit annoying at times for users.

I'd much rather have an unneeded `cliOptions`, than need to add a new kind of option and have no place to put it :-)